scripts/yara_fn/yara_fn.py

```diff
@@ -113,15 +113,15 @@ def get_basic_block_rule(bb):
                     masked_bytes.append('??')
                 else:
                     bytes.append(byte)
-                    masked_bytes.append('%02X' % (byte))
+                    masked_bytes.append('%02x' % int(byte.encode('hex'), 16))
```

Comment:
this looks a bit roundabout. could the same be accomplished like:  `(ord(byte))` ?

i'm also hesitant about py2/py3 here. the proposed solution looks like its a py2 construct. would you add a version switch here to handle both py2 and py3?